utils/blockbuilder/templates/blockname.grc.j2

```diff
@@ -97,16 +97,17 @@ inputs:
     {%if 'multiplicity' in port %}multiplicity: {{port['multiplicity']|grc_type(ref=True)}}{% endif %}
     label: {{port['id']}}
     optional: {{ port['optional'] if 'optional' in port else 'false' }}
-{% endif %}{% endfor %}
+{% endfor %}
 -   domain: message
     id: param_update
     label: param
     optional: true
     hide: ${ not showports }
 
+{% for port in ports if port['direction'] == 'output'%}
+{% if loop.first %}
```

Comment:
Should we do the same for input ports?

Reply:
This was my first thought, too. But then I noticed that input ports are never empty
https://github.com/gnuradio/gnuradio/blob/64951620164f098d9c16f5bc47bc562219df0606/utils/blockbuilder/templates/blockname.grc.j2#L88
.......
https://github.com/gnuradio/gnuradio/blob/64951620164f098d9c16f5bc47bc562219df0606/utils/blockbuilder/templates/blockname.grc.j2#L101-L105

Reply:
oh that's right.  i forgot about the default message ports.